src/Uno.UI/ts/Interop/ManagedObject.ts

```diff
@@ -4,7 +4,7 @@
 		private static dispatchMethod: (handle: string, method: string, parameters: string) => number;
 
 		private static init() {
-			ManagedObject.dispatchMethod = (<any>Module).mono_bind_static_method("[Uno.Foundation.Runtime.Wasm] Uno.Foundation.Interop.JSObject:Dispatch");
+			ManagedObject.dispatchMethod = (<any>Module).mono_bind_static_method("[Uno.Foundation.Runtime.WebAssembly] Uno.Foundation.Interop.JSObject:Dispatch");
```

Comment:
Codacy found an issue: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5894286)